packages/build-info/src/project.ts

```diff
@@ -185,17 +185,14 @@ export class Project {
   }
 
   /** Retrieves the package.json and if one found with it's pkgPath */
-  async getPackageJSON(startDirectory?: string): Promise<PackageJson & { pkgPath: string | null }> {
+  async getPackageJSON(startDirectory?: string): Promise<Partial<PackageJson> & { pkgPath: string | null }> {
     const pkgPath = await this.fs.findUp('package.json', {
       cwd: startDirectory || this.baseDirectory,
       stopAt: this.root,
     })
     if (pkgPath) {
       const json = await this.fs.readJSON<PackageJson>(pkgPath)
```

Comment:
For anyone else reading this by any chance... I was confused by this PR until I pulled the code down and poked it around. For posterity, all is explained by this `fs.readJSON<T>` helper returning `Partial<T>` for some reason!